### PR TITLE
[samplecode]fix: add request with default value in regular paged callable method

### PR DESCRIPTION
**Pull request by summer-ji-eng**
fix issue #680

### CODE REVIEW

**Review by miraleung — APPROVED** [review]

File: test/integration/goldens/logging/LoggingClient.java [hunk 1]
```diff
@@ -613,6 +613,14 @@ public final ListLogEntriesPagedResponse listLogEntries(ListLogEntriesRequest re
    *
    * <pre>{@code
    * try (LoggingClient loggingClient = LoggingClient.create()) {
+   *   ListLogEntriesRequest request =
+   *       ListLogEntriesRequest.newBuilder()
+   *           .addAllResourceNames(new ArrayList<String>())
+   *           .setFilter("filter-1274492040")
+   *           .setOrderBy("orderBy-1207110587")
+   *           .setPageSize(883849137)
+   *           .setPageToken("pageToken873572522")
```
Comment (vam-google) [inline test/integration/goldens/logging/LoggingClient.java:622]: Setting page token in the initial request seems harmful before you start paginated call there is no page token (i.e. page token makes sense only for pages 2+, but the first page does not have page token).

File: test/integration/goldens/logging/LoggingClient.java [hunk 2]
```diff
@@ -613,6 +613,14 @@ public final ListLogEntriesPagedResponse listLogEntries(ListLogEntriesRequest re
    *
    * <pre>{@code
    * try (LoggingClient loggingClient = LoggingClient.create()) {
+   *   ListLogEntriesRequest request =
+   *       ListLogEntriesRequest.newBuilder()
+   *           .addAllResourceNames(new ArrayList<String>())
+   *           .setFilter("filter-1274492040")
+   *           .setOrderBy("orderBy-1207110587")
+   *           .setPageSize(883849137)
```
Comment (vam-google) [inline test/integration/goldens/logging/LoggingClient.java:621]: Can we assing something reasonable (i.e smaller, like 10 or 100) to these kind of fields (random huge number for page size is confusing and not realistic)?

**Review by vam-google — COMMENTED** [review]
LGTM, but I was wondering how difficult would it be to address those the two usability-related comments?

File: test/integration/goldens/logging/LoggingClient.java (hunk 2 above)
Comment (summer-ji-eng) [reply test/integration/goldens/logging/LoggingClient.java:621]: Currently, we use hashcode of field' name. [source code](https://github.com/googleapis/gapic-generator-java/blob/c5a32cc3fe74a62898228f0c39a455e7701d2f1c/src/main/java/com/google/api/generator/gapic/composer/defaultvalue/DefaultValueComposer.java#L133-L139)

IMO, smaller number like 10 make more sense in reality. And PageSize is fixed filed to be set for default request variable. Looks like it is doable in code. @miraleung what is your opinion?

File: test/integration/goldens/logging/LoggingClient.java (hunk 1 above)
Comment (summer-ji-eng) [reply test/integration/goldens/logging/LoggingClient.java:622]: FYI @miraleung

**Pull request merged**